src/sentry/utils/snuba.py

```diff
@@ -866,14 +866,18 @@ def _snql_dryrun_query(params: Tuple[SnubaQuery, Hub, Mapping[str, str]]) -> Raw
     query_params, forward, reverse = query_data
     og_debug = query_params.get("debug", False)
     referrer = headers.get("referer", "<unknown>")
+    snql_entity = headers.get("snql_entity")
```

Comment:
Do we still want this in the headers after this point?

Reply:
I forgot about this, I'll remove the header.